site/docs/how-to-vote.md

```diff
@@ -0,0 +1,74 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Voting on an Iceberg Release
+
+Before a new version of Apache Iceberg is released, a release candidate will be prepared and a vote
+will occur. Announcements will be made on the Iceberg [developer's mailing list](http://127.0.0.1:8000/#community/#mailing-lists)
+and will include links to the following:
+
+- A source tarball
+- A signature (.asc)
+- A checksum (.sha512)
+- Key files
+- GitHub change comparison
+
+After downloading the source tarball, signature, checksum, and key files, follow
+these steps to participate in the voting process.
+
+Import the keys.
+```bash
+gpg --import ${PATH_TO_KEY_FILES}
+```
+
+Verify the signature.
+```bash
+gpg --verify apache-iceberg-${VERSION}.tar.gz.asc
+```
+
+Verify the checksum.
+```bash
+sha512sum -c apache-iceberg-${VERSION}.tar.gz.sha512
```

Comment:
On my MacBook, I don't have sha512sum. Maybe others do come with that by default, but I've been using `shasum -a 512` which is equivalent.

If others MacBooks have that (and they didn't install it themselves), ignore this. Otherwise we might consider changing it.

Reply:
No you're right, it doesn't exist by default on mac, you have to install coreutils. I'll make a note of that here.

Reply:
Updated with this note
```md
**note**: If you're on a mac that does not have coreutils installed, you can use `shasum -a 512` instead.
```

Reply:
Can we give the `shasum -a 512` command instead? I have shasum on my linux machine so we probably just need that one.

Reply:
Updated to use the `shasum -a 512` command